libwebthree/WebThree.h

```diff
@@ -128,7 +128,7 @@ class WebThreeDirect: public NetworkFace
         bytesConstRef _network = bytesConstRef(), bool _testing = false);
```

Comment:
Why does this still have `bzz` and `shh` in the `_interfaces` above?

Reply:
Yeah, they can be deleted, they are ignored in `WebThreeDirect` constructor implementation anyway